apps/client/src/pages/dashboard/apis/categoryApi.ts

```
@@ -0,0 +1,21 @@
+
+
+export const getDashboardCategories = async (): Promise<
+  ApiResponse<CategoryListResponse>
+> => {
+  try {
+    const { data } = await apiRequest.get<ApiResponse<CategoryListResponse>>(
+      '/api/v1/categories/dashboard'
+    );
+    return data;
+  } catch (error: any) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)